MainWindow.cpp

```diff
@@ -167,13 +220,14 @@ void MainWindow::CreateWidgets()
 
     galaxyView = new GalaxyView(map, tabs, tabs);
 
+    // Initialize the sidebar with the system, government, fleet, trade, and minables data.
     detailView = new DetailView(map, galaxyView, box);
     detailView->setMinimumWidth(300);
     detailView->setMaximumWidth(300);
     galaxyView->SetDetailView(detailView);
 
     systemView = new SystemView(map, detailView, tabs, tabs);
-    auto it = map.Systems().find("Sol");
+    const auto &it = map.Systems().find("Sol");
```

Comment:
Same here, it makes more sense to just move-construct the iterator returned by find() rather than taking a reference to it.